docs: add uhoo integration documentation (#42581)
home-assistant/home-assistant.github.io: source/_integrations/uhooair.markdown

---
title: uHoo
description: Instructions on how to integrate uHoo with Home Assistant

> ha_release: 2025.3

Reviewer: ```suggestion
ha_release: 2026.1
```

> ha_release: 2026.1

Reviewer 2: ```suggestion
ha_release: 2026.2
```

Author: Noted

ha_iot_class: Cloud Polling
ha_codeowners:
  - '@getuhoo'
  - '@joshsmonta'
ha_domain: uhooair

> ha_integration_type: iot-cloud

Reviewer 2: iot-cloud isn't an option here

Author: Okay, I'll just put in the default `hub`

---

The **uHoo** {% term integration %} lets you integrate [uHoo](https://getuhoo.com) devices into Home Assistant.

## About uHoo

uHoo delivers advanced, real-time indoor air quality monitoring designed to help you understand, manage, and improve the air in your home. By measuring 9 key environmental factors, uHoo provides accurate and science-backed insights that support healthier living and smarter home automation.

What sets uHoo apart is its suite of proprietary health and safety indices that go beyond raw data to show you exactly how your environment affects your well-being, helping you act before problems even start.
- Virus index
- Mold index
- Influenza index

Connecting uHoo with Home Assistant unlocks even more possibilities: create air-quality-based routines, receive instant alerts, track long-term trends, and maintain a safer, more energy-efficient home environment.

With uHoo, you can confidently make better decisions about your indoor air and stay on top of your game every day.

> ## Supported Devices

Reviewer: ```suggestion
## Supported devices
```

---
title: uHoo
description: Instructions on how to integrate uHoo with Home Assistant
ha_release: 2025.3
ha_iot_class: Cloud Polling
ha_codeowners:
  - '@getuhoo'
  - '@joshsmonta'
ha_domain: uhooair
ha_integration_type: iot-cloud
related:
  - url: https://developers.home-assistant.io/docs/documenting/standards
    title: Documentation standard
  - url: https://developers.home-assistant.io/docs/core/integration-quality-scale/rules/
    title: Integration Quality Scale - Rules
  - docs: /docs/glossary/
    title: Glossary
  - docs: /docs/tools/quick-bar/#my-links

>     title: My link

Reviewer: ```suggestion
```
As this is the user documentation, we don't link from here to the developer documentation.

Author: This is noted.

---

<!--- The integration documentation template provides a documentation structure as well as some example content per section. The example content is meant for inspiration, it may not apply for your integration or will at least have to be adapted. -->

> <!--- Use this template together with the developer documentation, under [Documentation standard](https://developers.home-assistant.io/docs/documenting/standards) and the documentation rules of the [Integration Quality Scale](https://developers.home-assistant.io/docs/core/integration-quality-scale/rules/). -->

Reviewer: ```suggestion
```
Please remove comments

> The **uHoo** {% term integration %} supports the integration of [uHoo](https://getuhoo.com) devices.

Reviewer: ```suggestion
The **uHoo** {% term integration %} lets you integrate [uHoo](https://getuhoo.com) devices into Home Assistant.
```
Maybe something like this?

- Virus Index
- Mold Index
- Influenza Index

Connecting uHoo with Home Assistant unlocks even more possibilities: create air-quality-based routines, receive instant alerts, track long-term trends, and maintain a safer, more energy-efficient home environment.

With uHoo, you can confidently make better decisions about your indoor air and stay on top of your game every day.

## Supported Devices

- [uHoo Smart Air Monitor](https://getuhoo.com/home/smart-air-monitor)
- [uHoo Caeli Air Monitor](https://getuhoo.com/home/caeli)

## Prerequisites

1. Download the **uHoo** app from the [App Store](https://apps.apple.com/us/app/uhoo/id1084953997) or [Google Play](https://play.google.com/store/apps/details?id=com.uhooair).
2. Sign up for a uHoo account.
3. Register at least one device to your account.
4. Upgrade your account to uHoo Premium.
5. Login the [uHoo web dashboard](https://premium.getuhoo.com/) and tap the **Account** icon in the bottom left corner.
6. Navigate to the **API Key** section and enter your account password to create an API key.

{% include integrations/config_flow.md %}

## Supported functionality

### Sensors

- Temperature

> - Relative Humidity

Reviewer: ```suggestion
- Relative humidity
```
We use [sentence-style capitalization](https://learn.microsoft.com/en-us/style-guide/capitalization), also in UI strings. Please fix also for the others.

- PM2.5 (Particulate Matter)
- Total Volatile Organic Compounds (TVOC)
- Carbon Dioxide (CO2)
- Carbon Monoxide (CO)
- Air Pressure
- Ozone
- Nitrogen Dioxide (NO2)

### Indices

- Virus Index
- Mold Index

### Data frequency

> - This integration {% term polling polls %} data from the device(s) every 5 minutes.

Reviewer: ```suggestion
- This integration {% term polling polls %} data from the devices every 5 minutes.
```
